python/model.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+
+import argparse
+from lib.database import *
+from lib.tests import *
+from lib.colors import colors
+import random
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Database setup and testing script")
+    parser.add_argument(
+        "-q", "--quick", action="store_true", help="Perform quick setup"
+    )
```

Comment:
You can use this to only load 10K of the seed rows, if you're looking for a quick turn around as you develop or test.

Reply:
You bet John. I have spun up a `t3.medium` on EC2 so I can test these all on the same intel machine, and I'll be happy to benchmark mine on the same box. Thank you for asking!

//cc @mddilley @roseeichelmann